.github/workflows/build-testing.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     with:
       arch: amd64
       tag: testing-amd64
-      core_ref: v19.13.0
+      core_ref: v19.14.0
```

Comment:
Apologies, I missed this on the first look. The testing version update here is going to conflict with:
- #494

Reply:
np, reverted all testing changes.